evacuees/src/API/EMBC.Registrants.API/EvacuationsModule/EvacuationAutoMapperProfile.cs

```diff
@@ -45,7 +47,9 @@ public EvacuationAutoMapperProfile()
                 .ForPath(d => d.era_postalcode, opts => opts.MapFrom(s => s.EvacuatedFromAddress.PostalCode))
                 .ForPath(d => d.era_country, opts => opts.MapFrom(s => s.EvacuatedFromAddress.Country.Name))
                 .ForPath(d => d.era_province, opts => opts.MapFrom(s => s.EvacuatedFromAddress.StateProvince.Name))
-                .ForPath(d => d.era_Jurisdiction, opts => opts.MapFrom(s => s.EvacuatedFromAddress.Jurisdiction));
+                .ForPath(d => d.era_Jurisdiction, opts => opts.MapFrom(s => s.EvacuatedFromAddress.Jurisdiction))
+                .ForPath(d => d.era_Jurisdiction.era_RelatedProvinceState.era_code, opts => opts.MapFrom(s => s.EvacuatedFromAddress.Jurisdiction))
+                .ForPath(d => d.era_Jurisdiction.era_RelatedProvinceState.era_RelatedCountry.era_countrycode, opts => opts.MapFrom(s => s.EvacuatedFromAddress.Jurisdiction.CountryCode));
```

Comment:
This is probably not going to work, v9.0 doesn't support multi level traversal AFAIK

Reply:
Fair enough. I'll get rid of it.

Reply:
Done!